boards/hardkernel/odroid_go/odroid_go_esp32_procpu.dts

```diff
@@ -142,17 +142,6 @@
 	status = "okay";
 	pinctrl-0 = <&spim3_default>;
 	pinctrl-names = "default";
-
-	sdhc0: sdhc@1 {
```

Comment:
Perhaps I'm missing something- should an SDHC definition be added for this board to enable the controller? Or does this board's DTS include the esp_wrover_kit somehow?